base/random.jl

```diff
@@ -1096,7 +1096,13 @@ const ziggurat_nor_r      = 3.6541528853610087963519472518
 const ziggurat_nor_inv_r  = inv(ziggurat_nor_r)
 const ziggurat_exp_r      = 7.6971174701310497140446280481
 
+"""
+    randn([rng], [T], [dims...])
 
+Generate a normally-distributed random number of type `T` with mean 0 and standard deviation 1.
+The type `T` can be `Float16`, `Float32`, or `Float64` (the default).
```

Comment:
I would move this to the end of the docstring and change it to: 
"The `Base` module currently provides an implementation for the types `Float16`, `Float32`, and `Float64` (the default)". Any comment?